src/main/java/com/leinardi/pycharm/mypy/MypyAnnotator.java

```diff
@@ -91,7 +124,10 @@ private MypyPlugin plugin(final Project project) {
             if (map.isEmpty()) {
                 return NO_PROBLEMS_FOUND;
             }
-            return map.get(psiFile);
+
+            long duration = System.currentTimeMillis() - startTime;
+            LOG.debug("Mypy scan completed: " + psiFile.getName() + " in " + duration + " ms");
```

Comment:
I added several new logging calls when I was working on this. Should I keep them in?

Reply:
Since the log level is debug, I don't mind too much. It's up to you if you want to keep them or not :+1: